### DISCUSSION TITLE
Draggable iFrame

### DISCUSSION THREAD

**Discussion by raunakdoesdev** [xyflow/xyflow#3100] [Q&A]
One of the nodes I'm rendering is an iFrame. Unfortunately, it's not able to drag easily. I'd like to be able to propagate the drag events from the iFrame to the Reactflow graph. I'd also like to be able to specify parts of the iFrame as nodrag, but I'm not sure if this will work due to the sandboxing of the iFrame. Any suggestions or pointers here?

**Comment by ghost** [comment]
Any updates here?
